CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Change Log
```

Comment:
The changelog can probably be removed again. I created it more for myself to keep track of things...